modular_skyrat/modules/customization/modules/language/_language_holder.dm

```diff
@@ -0,0 +1,81 @@
+/datum/language_holder/vox
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/vox = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/vox = list(LANGUAGE_ATOM)
+	)
+
+/datum/language_holder/skrell
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/skrell = list(LANGUAGE_ATOM),
+		/datum/language/slime = list(LANGUAGE_ATOM)
+	)
+
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/skrell = list(LANGUAGE_ATOM),
+		/datum/language/slime = list(LANGUAGE_ATOM)
+	)
+
+/datum/language_holder/shadowpeople
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/voltaic = list(LANGUAGE_ATOM),
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/voltaic = list(LANGUAGE_ATOM),
+	)
+
+/datum/language_holder/skeleton
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/terrum = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/terrum = list(LANGUAGE_ATOM)
+	)
+
+/datum/language_holder/golem/bone
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/terrum = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/terrum = list(LANGUAGE_ATOM)
+	)
+
+/datum/language_holder/felinid
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/yangyu = list(LANGUAGE_ATOM)
+	)
+	spoken_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/yangyu = list(LANGUAGE_ATOM)
+	)
+
+/datum/language_holder/monkey
+	understood_languages = list(
+		/datum/language/common = list(LANGUAGE_ATOM),
+		/datum/language/uncommon = list(LANGUAGE_ATOM)
```

Comment:
don't like this one at all

Reply:
Ditto

Reply:
what was xyel's reasoning for monkeys not talking... monkey?

Reply:
Wait, no, nevermind, I forgot I set monkeys to speak uncommon by default as a test, and forgot to revert due to shortcomings.